src/System.Drawing.Common/tests/Text/PrivateFontCollectionTests.cs

```diff
@@ -129,7 +129,6 @@ public void AddFontFile_NoSuchFilePath_ThrowsFileNotFoundException()
         }
 
         [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsNotWindowsNanoServer))]
-        [ActiveIssue("https://github.com/dotnet/corefx/issues/8655")]
         public void AddFontFile_LongFilePath_ThrowsException()
         {
             using (var fontCollection = new PrivateFontCollection())
```

Comment:
the if-else code below could use `AssertExtensions.Throws<FileNotFoundException, PathTooLongException>(...)`